content/docs/command-reference/version.md

```diff
@@ -14,18 +14,16 @@ usage: dvc version [-h] [-q | -v]
-| [`DVC version`](#components-of-dvc-version) | Version of DVC (along with a Git commit hash if running a development version)                                                                                        |
-| `Python version`                            | Version of Python used in the environment where DVC is initialized                                                                                                    |
-| `Platform`                                  | Information about the operating system of the machine                                                                                                                 |
-| [`Binary`](#what-we-mean-by-binary)         | Shows whether DVC was installed from a package or from a binary release                                                                                               |
-| `Package`                                   | Name of the package manager used to install DVC if any (`pip`, `conda`, etc)                                                                                          |
-| `Supported remotes`                         | Remote types that have all needed dependencies installed for                                                                                                          |
-| `Cache`                                     | [Type of links](/doc/user-guide/large-dataset-optimization#file-link-types-for-the-dvc-cache) supported between the <abbr>workspace</abbr> and the <abbr>cache</abbr> |
-| `Filesystem type`                           | Shows the filesystem type (eg. ext4, FAT, etc.) and mount point of the cache and <abbr>workspace</abbr> directories                                                   |
+| [`DVC version`](#components-of-dvc-version) | Version of DVC (along with a Git commit hash in case of a development version) and name of the package manager (`pip`, `conda`, etc.) or binary used to install DVC   |
+| `Platform`                                  | Version of Python in the environment where DVC is initialized, and information about the operating system                                                             |
+| `Supports`                                  | Shows remote types that have all needed dependencies installed                                                                                                        |
+| `Cache types`                               | [Type of links](/doc/user-guide/large-dataset-optimization#file-link-types-for-the-dvc-cache) supported (between <abbr>workspace</abbr> and <abbr>cache</abbr>        |
+| `Cache directory`                           | Shows the filesystem type (eg. ext4, FAT, etc.) and mount point of the <abbr>cache</abbr> directory                                                                   |
```

Comment:
Please let's thing through text, don't just copy/paste. Thanks:

- No need for "Shows the"
- eg. -> e.g.
- Is "mount point" a well-known term? (I honestly don't know.)